From: Jasem Mutlaq <mutlaqja@ikarustech.com>
Date: Wed, 30 Dec 2015 17:02:20 +0000
Subject: Removing unncessary dependencies on Qt modules for Test OpenGL QML QtQuick QtMultimedia. We should re-enable QML/QtQuick if someone manager to migrate what is interesting tonight tool
X-Git-Url: http://quickgit.kde.org/?p=kstars.git&a=commitdiff&h=2f09f40aca053048ffb65621e3c18780cdc5cbe3
---
Removing unncessary dependencies on Qt modules for Test OpenGL QML QtQuick QtMultimedia. We should re-enable QML/QtQuick if someone manager to migrate what is interesting tonight tool
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,8 @@
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules" ${CMAKE_MODULE_PATH})
 
-find_package(Qt5 5.2 REQUIRED COMPONENTS Gui Qml Quick Xml Sql Svg OpenGL PrintSupport Multimedia Test)
+#find_package(Qt5 5.2 REQUIRED COMPONENTS Gui Qml Quick Xml Sql Svg PrintSupport)
+find_package(Qt5 5.2 REQUIRED COMPONENTS Gui Xml Sql Svg PrintSupport)
 include(KDEInstallDirs)
 include(KDECompilerSettings)
 include(KDECMakeSettings)
--- a/kstars/CMakeLists.txt
+++ b/kstars/CMakeLists.txt
@@ -549,9 +549,8 @@
         Qt5::PrintSupport
         Qt5::Sql
         Qt5::Svg
-        Qt5::Multimedia
-        Qt5::Qml
-        Qt5::Quick
+        #Qt5::Qml
+        #Qt5::Quick
 	${ZLIB_LIBRARIES}
         )
 
--- a/kstars/kstars.cpp
+++ b/kstars/kstars.cpp
@@ -39,7 +39,7 @@
 #include "dialogs/finddialog.h"
 #include "dialogs/exportimagedialog.h"
 #include "observinglist.h"
-#include "whatsinteresting/wiview.h"
+//#include "whatsinteresting/wiview.h"
 
 #include "kstarsadaptor.h"
 
--- a/kstars/kstarsactions.cpp
+++ b/kstars/kstarsactions.cpp
@@ -30,8 +30,8 @@
 #include <QDockWidget>
 #include <QPointer>
 #include <QInputDialog>
-#include <QQuickWindow>
-#include <QQuickView>
+//#include <QQuickWindow>
+//#include <QQuickView>
 #include <QDebug>
 #include <QAction>
 #include <QFileDialog>

